From 2c568ed308f23b7dabebef8ef674ba5bad023183 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Sat, 30 Jul 2011 22:41:01 +0800
Subject: [PATCH] Revert "Bluetooth: Increase timeout for legacy pairing from
 10 seconds to 40 seconds."

This reverts commit 1f39bbdd42999892f5f6ef8aeb03b3fff6d1ebd6.
---
 include/net/bluetooth/hci_core.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/bluetooth/hci_core.h b/include/net/bluetooth/hci_core.h
index b6d36cbb2773..7b640aeddb64 100644
--- a/include/net/bluetooth/hci_core.h
+++ b/include/net/bluetooth/hci_core.h
@@ -358,7 +358,7 @@ static inline void hci_conn_put(struct hci_conn *conn)
 			if (conn->state == BT_CONNECTED) {
 				timeo = msecs_to_jiffies(conn->disc_timeout);
 				if (!conn->out)
-					timeo *= 20;
+					timeo *= 2;
 			} else
 				timeo = msecs_to_jiffies(10);
 		} else
-- 
2.35.3

